sticker-graph/sticker_graph/__init__.py

```diff
@@ -0,0 +1,3 @@
+from sticker_graph.conv_model import ConvModel
```

Comment:
These are unused, in the `write-scripts` the imports are absolute. Is this intended for possible library use? In that case, we should provide more pydoc on the `shapes` and `config` parameters of the models.

Reply:
> These are unused, in the write-scripts the imports are absolute.

No? E.g. in `sticker-write-rnn-graph`:

from sticker_graph import RNNModel

Reply:
Oh, my bad. I checked `rnn_model.py` and `conv_model.py` for imports and mistook the absolute imports of `Model` there. 

This seems fine then. Although, it still invites to (undocumented) library use.

Reply:
Indeed, it does. I don't know if we actually want to provide this as a public API. My motivation for separating the module was so that we can make distro packages, where the scripts have to live in `$PREFIX/bin` and the remaining Python files in some module directory.

Maybe we could add some remark that the API is not documented, because it is not intended to be a public API?

Reply:
A start could be to remove the imports from `__init__.py` and replace the ones in the script by absolute imports. Would this clash with one of your envisioned use cases?

Reply:
Done!